api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -76,18 +179,31 @@
   }
 
   @Override
-  public ResponseEntity<CreateAccountResponse> createAccount(CreateAccountRequest request) {
+  public ResponseEntity<Profile> createAccount(CreateAccountRequest request) {
     if (request.getInvitationKey() == null
         || !request.getInvitationKey().equals(cloudStorageService.readInvitationKey())) {
-      return ResponseEntity.badRequest()
-          .body(new CreateAccountResponse().message(
-              "Missing or incorrect invitationKey (this API is not yet publicly launched)"));
+      throw new BadRequestException(
+          "Missing or incorrect invitationKey (this API is not yet publicly launched)");
     }
-    directoryService.createUser(
+    com.google.api.services.admin.directory.model.User googleUser = directoryService.createUser(
         request.getGivenName(), request.getFamilyName(), request.getUsername(),
         request.getPassword()
     );
-    return ResponseEntity.status(HttpStatus.CREATED).build();
+
+    // Create a user that has no data access or FC user associated.
+    User user = new User();
```

Comment:
Suggested comments: We don't want to join with DS data. At some point we'll want bring-your-own-account functionality. Keeping in sync: Possibly do so asynchronously.